src/Microsoft.Windows.CsWin32/templates/CharSet_Workaround.cs

```diff
@@ -1,4 +1,4 @@
-﻿namespace Windows.Win32.System.Runtime.InteropServices
+﻿namespace System.Runtime.InteropServices
```

Comment:
Does this not conflict with https://learn.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.charset?view=net-9.0